pact-tests/Pact/Core/Test/ReplTests.hs

```diff
@@ -89,7 +89,7 @@ runReplTest (ReplSourceDir path) pdb file src interp = do
       let rendered = renderLocatedPactErrorFromState rstate' e
       assertFailure (T.unpack rendered)
     Right _ -> do
-      traverse_ ensurePassing . _replTestResults =<< readIORef stateRef
+      traverse_ ensurePassing . reverse . _replTestResults =<< readIORef stateRef
```

Comment:
Btw this reverse is just a QOL fix.

I noticed when debugging values by running `cabal run tests -- -p 'coin-v5'` that the errors being spit out by the repl tests were in reverse order.